queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT date_of_birth FROM animals WHERE date_of_birth BETWEEN '2016/01/01' AND '2019/12/31';
+SELECT escape_attempts FROM animals WHERE neutered = false AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 OR weight_kg <= 17.3;
```

Comment:
- [ ] Line 10 query returns all animals with a weight between 10.4kg or 17.3kg. However, we are asked to find all animals with a weight between 10.4kg and 17.3kg. A good query would be `SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
`